lib/logging/meson.build

```diff
@@ -0,0 +1,25 @@
+subdir('include')
+subdir('lib')
+
+hse_logging = static_library(
+    '@0@-logging'.format(meson.project_name()),
+    hse_logging_sources,
+    build_config_h,
+    include_directories: [
+        hse_logging_includes,
+        component_root_includes,
+        # TODO: Remove when compiler.h moves around
+        ikvdb_includes,
+    ],
+    dependencies: [
+        cjson_dep,
+        hse_error_dep,
+        rbtree_dep,
```

Comment:
Seems like these dependencies are incorrect.

Reply:
I wish they were incorrect. It has to do with the leaks of hse_util. `error/merr.h` needs `hse_util/event_counter.h` which then brings in the entire kitchen sink. Trust me, I hate this problem and why I have been an advocate for re-organizing our files to create proper dependency chains.

Reply:
I got rid of `ikvdb_includes` and ended up with something quite nice :)